src/EditorFeatures/Test/CodeGeneration/CodeGenerationTests.cs

```diff
@@ -591,14 +591,14 @@ public partial class CodeGenerationTests
         internal static Func<SemanticModel, IParameterSymbol> Parameter(Type type, string name, bool hasDefaultValue = false, object defaultValue = null, bool isParams = false)
         {
             return s => CodeGenerationSymbolFactory.CreateParameterSymbol(
-                default(ImmutableArray<AttributeData>), RefKind.None, isParams, GetTypeSymbol(s.Compilation, type), name,
+                default, RefKind.None, isParams, GetTypeSymbol(s.Compilation, type), name,
                 isOptional: hasDefaultValue, hasDefaultValue: hasDefaultValue, defaultValue: defaultValue);
         }
 
         internal static Func<SemanticModel, IParameterSymbol> Parameter(string typeFullName, string parameterName, bool hasDefaultValue = false, object defaultValue = null, bool isParams = false, int typeArrayRank = 0)
         {
             return s => CodeGenerationSymbolFactory.CreateParameterSymbol(
-                default(ImmutableArray<AttributeData>), RefKind.None, isParams, GetTypeSymbol(s.Compilation, typeFullName, typeArrayRank), parameterName,
+                default, RefKind.None, isParams, GetTypeSymbol(s.Compilation, typeFullName, typeArrayRank), parameterName,
```

Comment:
Not against this, but I do have to admit the old form actually was clearer: it was clear we were creating a parameter with no attributes, as opposed to, well, now it's a default...something?

Reply:
will fix up!

Reply:
Have tweaked things.